src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActor.cs

```diff
@@ -23,7 +23,7 @@ namespace Akka.Streams.Kafka.Stages.Consumers.Actors
     /// </summary>
     /// <typeparam name="K">Message key type</typeparam>
     /// <typeparam name="V">Message value type</typeparam>
-    internal class KafkaConsumerActor<K, V> : ActorBase
+    internal class KafkaConsumerActor<K, V> : ActorBase, ILogReceive
```

Comment:
Added `ILogReceive` to the internal consumer actor for debugging purposes. Won't do anything unless `akka.actor.debug.receive = on` and `akka.loglevel = DEBUG`